test/scripts/run-tests.sh

```diff
@@ -54,11 +54,13 @@ cd ${GO_DIR}
 ./deploy.sh
 
 TIMEOUT=60
-until kubectl get pods -n katib | grep 1/1 | [[ $(wc -l) -eq 7 ]]; do
+PODNUM=$(kubectl get pods -n katib | grep -v NAME | wc -l)
```

Comment:
You can avoid grep like this but I don't think it matters much `kubectl get pods -o jsonpath='{.items[*].metadata.name}' | wc -w`